src/games/GameDialogFileList.tsx

```diff
@@ -216,7 +216,6 @@ interface SortableItemProps {
 
 const SortableItem: React.FC<SortableItemProps> = (props) => {
   const theme = useTheme()
-  const canSort = !props.file.isIwad && props.file.selected
```

Comment:
Turns out there's not much gained from disabling sorting, and it makes things confusing when an IWAD has multiple files.

Two things help with this:

- Now, the first file from a game that's labelled as an IWAD is treated as the IWAD, otherwise it's added as a regular file
- The user can sort all files including IWADs, placing the real IWAD at the top which is a sane thing to do